mac/KeymanEngine4Mac/KeymanEngine4MacTests/KMEngineTests.m

```diff
@@ -553,21 +536,40 @@ - (void)testCoreProcessEvent_eventForFWithElNuerKmx_ReturnsCorrectCharacter {
     KMXFile *kmxFile = [KeymanEngineTestsStaticHelperMethods getKmxFileForElNuerTests];
     KMEngine *engine = [[KMEngine alloc] initWithKMX:kmxFile context:@"" verboseLogging:YES];
     NSEvent *event = [NSEvent keyEventWithType:NSEventTypeKeyDown location:NSMakePoint(0, 0) modifierFlags:0 timestamp:0 windowNumber:0 context:nil characters:@"f" charactersIgnoringModifiers:@"f" isARepeat:NO keyCode:kVK_ANSI_F];
-    NSArray *actions = [engine processEvent:event];
-    XCTAssert(actions.count == 1, @"Expected one action");
-    CoreAction *action = actions[0];
-    XCTAssert([action isCharacter], @"Expected CharacterAction");
-    XCTAssert([action.content isEqualToString:@"ɣ"], @"Expected output to be 'ɣ'.");
+    CoreKeyOutput *output = [engine processEvent:event];
+    XCTAssert([output.textToInsert isEqualToString:@"ɣ"], @"Expected output to be 'ɣ'.");
+    XCTAssert(!output.hasCodePointsToDelete, @"expected to delete nothing");
+    XCTAssert(!output.emitKeystroke, @"expected to emit nothing");
 }
 
-- (void)testCoreProcessEvent_eventDeleteWithElNuerKmx_EmptiesContextReturnsDelete {
+// TODO: enable after resolving issue getting options from Keyman Core
```

Comment:
Are you tracking this?

Reply:
Yes, I will enable after merging to ensure that the core changes cause this test to pass.